docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -152,13 +171,17 @@ The minimum supported Node.js version has been changed from 12.0.0 to 14.0.0, si
     }}
   />
   ```
-
+- The `componentError` event was removed. Use the [error boundary](https://reactjs.org/docs/error-boundaries.html) to catch errors thrown during rendering.
 - The `GridCallbackDetails['api']` was removed from event details. Use the `apiRef` returned by `useGridApiContext` or `useGridApiRef` instead.
 - The `cellFocusIn` and `cellFocusOut` events are internal now. Use `componentsProps.cell.onFocus` and `componentsProps.cell.onBlur` props instead.
 
+:::info
+To know more about the supported events and their signatures, check the [events catalog](https://next.mui.com/x/react-data-grid/events/#catalog-of-events) page.
+:::
+
 ### Columns
 
-- The `GridColDef['hide']` property was removed. Use `GridColDef['columnVisibility']` instead.
```

Comment:
I couldn't find `GridColDef['columnVisibility']`, probably it's removed?

Reply:
It has never existed 👌